examples/topics/Seattle_Lidar.ipynb

```diff
@@ -410,9 +409,16 @@
    "outputs": [],
    "source": [
     "illuminated = hillshade(raster.Z)\n",
-    "hv_shaded = hv.Image((raster.meterswest, raster.metersnorth, illuminated))\n",
-    "\n",
-    "tiles * rasterize(hv_shaded.options(opts.Image(cmap='greys', alpha=.8, height=600, width=600)))"
+    "hv_shaded = hv.Image((raster.meterswest, raster.metersnorth, illuminated))"
```

Comment:
Split into two cells since hillshading takes some time but the final plotting step is cheap and can easily be re-done with different options.